frontend/src/assets/styles/tokens.css

```
@@ -41,13 +49,44 @@
-  --surface-border: var(--surface-700);
+  --state-hover: color-mix(in srgb, var(--surface-400) 12%, transparent);
+  --state-active: color-mix(in srgb, var(--primary-color) 8%, transparent);
+  --surface-hover: var(--state-hover);
+  --surface-border: var(--border-subtle);
 
   /* Focus */
   --focus-ring-offset: 2px;
   --focus-ring: 0 0 0 0.2rem color-mix(in srgb, var(--primary-color) 40%, transparent);
 
   /* Typography */
   --font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.9.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)